data/phi0914/phi001/__cts__.xml

```diff
@@ -1,19 +1,15 @@
 <work xmlns="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi0914" projid="latinLit:phi001" urn="urn:cts:latinLit:phi0914.phi001" xml:lang="lat">
-    <title xml:lang="eng">The History of Rome</title>
     <title xml:lang="lat">Ab urbe condita</title>
     <translation projid="latinLit:perseus-eng3" urn="urn:cts:latinLit:phi0914.phi001.perseus-eng3" workUrn="urn:cts:latinLit:phi0914.phi001" xml:lang="eng">
         <label xml:lang="eng">The History of Rome</label>
-        <description xml:lang="eng">Perseus:bib:oclc,2311635, Livy. History of Rome. English
-            Translation by. Rev. Canon Roberts. New York, New York. E. P. Dutton and Co. 1912. 1.
-            Livy. History of Rome. English Translation. Rev. Canon Roberts. New York, New York. E. P.
-            Dutton and Co. 1912. 2.</description>
+        <description xml:lang="eng">Livy. History of Rome, Volumes 1-2. Roberts, Canon, Rev, translator. 
+            London, New York: J. M. Dent &amp; Sons; E. P. Dutton and Co., 1912.</description>
```

Comment:
I don’t think we use entities in metadata files as they do not display properly in SV, but I would have to find an example of this to be sure.

Reply:
You know I think we had a conversation about that, I can just change it.